Python/math/README.md

```diff
@@ -1 +1 @@
-a
+[Lucas Theorem](https://github.com/Rishikeshrajrxl/NeoAlgo/blob/rishikeshrajrxl/Python/math/Lucas%20Theorem.py)
```

Comment:
You need to update the main [README.md](https://github.com/TesseractCoding/NeoAlgo/blob/master/Python/README.md) of Python folder.